src/anomalib/engine/engine.py

```diff
@@ -975,6 +976,11 @@ def export(
         else:
             logging.error(f"Export type {export_type} is not supported yet.")
 
+        if compression_type and export_type != ExportType.OPENVINO:
```

Comment:
This modification is not needed as openvino export does this check internally. If something fails, it will fail before this line is called.